### PR TITLE
Tidy up and add comments to Package.swift

### PR DESCRIPTION
**Pull request by glbrntt**
Tidy up `Package.swift`:
- added comments to each target
- changed `Commander` from `.upToNextMinor(from: "0.8.0")` to `from: "0.8.0"`

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -41,46 +25,95 @@ let package = Package(
...
+    // GRPC dependencies:
+    // Main SwiftNIO package
+    .package(url: "https://github.com/apple/swift-nio.git", from: "2.2.0"),
+    // HTTP2 via SwiftNIO
+    .package(url: "https://github.com/apple/swift-nio-http2.git", from: "1.2.1"),
+    // TLS via SwiftNIO
+    .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "2.0.0"),
+
+    // Official SwiftProtobuf library, for [de]serializing data to send on the wire.
+    .package(url: "https://github.com/apple/swift-protobuf.git", from: "1.3.1"),
```
Comment (MrMage) [inline Package.swift]: How about "from 1.5.0"?

File: Package.swift [hunk 2]
```diff
@@ -41,46 +25,95 @@ let package = Package(
...
+    .target(
+      name: "protoc-gen-swiftgrpc",
+      dependencies: [
+        "SwiftProtobuf",
+        "SwiftProtobufPluginLibrary",
+        "protoc-gen-swift"
+      ]
+    ),
+
+    // Interoperability tests, this doesn't contain the CLI as this is reused
```
Comment (MrMage) [inline Package.swift]: What does the second "this" refer to? Could that "this" replaced for clarity?

File: Package.swift (hunk 1 above)
Comment (glbrntt) [reply Package.swift]: Ah yes, good idea!

File: Package.swift (hunk 2 above)
Comment (glbrntt) [reply Package.swift]: "this" refers to the target, will clarify!

Comment (MrMage) [reply Package.swift]: Sorry, mind further clarifying "this" to "the interoperability tests are..."=

**Comment by MrMage** [comment]
Thank you for the changes!

**Pull request merged**